driver/main.ml

```diff
@@ -142,6 +142,12 @@ module Options = Main_args.Make_bytecomp_options (struct
   let anonymous = anonymous
 end)
 
+let vmthread_deprecated_message = "\
+The -vmthread argument of ocamlc is deprecated\n\
+and will soon be abandoned. For VM-level scheduling, consider using a\n\
```

Comment:
isn't "soon abandoned" synonym with "deprecated" ?

Reply:
It's close enough to a synonym. "removed" would be a better word.

Reply:
What I meant is that one could simply say "VM-threads are deprecated".

Reply:
I guess the information that is missing is when the feature will indeed be removed. We don't have to specify it, but in general it is useful for users to know this.